cmd/livepeer/livepeer.go

```diff
@@ -287,6 +287,9 @@ func main() {
 	}
 
 	watcherErr := make(chan error)
+	var roundsWatcher *watchers.RoundsWatcher
```

Comment:
These variables can be declared in the else block below since we never need to use these variables when `*network == "offchain"`.

Reply:
These variables are also being used when setting up `DBOrchestratorPoolCache` at L648 , which is outside of the scope of the `if *network == "offchain {} else` block. 

We could move the lines that initiate `DBOrchestratorPoolCache`  but that requires shuffling some other conditionals. It also strips away one option for starting an orchestratorPool outside of the block that handles this logic, which wouldn't be very optimal in terms of readability.

Reply:
Oops missed this bit since I didn't unfold the code enough in the review comparison view - gotcha